wikidatarefisland/data_model/wikibase/value_matchers.py

```diff
@@ -69,7 +69,7 @@ def match_datetime(statement_reference):
         value = DateTimeValue(statement)
 
         # 11 - Day Precision, 9 - Year precision
-        if value.precision != '11' and value.precision != '9':
+        if value.precision != 11 and value.precision != 9:
```

Comment:
Only question I have here is if casting value.precision to an integer would be worthwhile? Just in case an alternative implementation of pipe 1 or 2 ends up serialising the precision as strings not integers. Or we get the item from a slightly different serialisation of the dumps

Reply:
Yes, that makes a whole lot of sense, thank you!